core/platform/datastore/gae_datastore_services.py

```diff
@@ -183,6 +210,7 @@ def any_of(*nodes):
 
 
 def make_cursor(urlsafe_cursor=None):
+    # type: (Optional[Text]) -> datastore_query.Cursor
```

Comment:
Yes. I'll start using str when we migrate to py3. Also can you please share the doc with me? Thanks!

Reply:
Yup, here it is: https://docs.google.com/document/d/100RBK4yGzA5-1a268FEDcRerlnkxwHsLkTMzAXAnyqM/edit . Note that this will be converted to a wiki page shortly.